Invoke-DiagnosticCheck.ps1

```diff
@@ -57,6 +59,7 @@ function Assert-AreDLLsPresent {
     # that application failed to load some DLL.
     $MissingDLLsErrorReturnCode = [int64]0xC0000135
     $InvalidImageFormatErrorReturnCode = [int64]0xC000007B
+    $SuccessReturnCode = [int64]0x00000000
```

Comment:
Nitpicky - all other Assert-* functions actually assert something. This function is not - therefore maybe let's rename it to Test-DllsArePresent or something?